luigi/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+    name: luigi
+    version: "2.0.0"
+
+source:
+    fn: luigi-2.0.0.tar.gz
+    url: https://pypi.python.org/packages/source/l/luigi/luigi-2.0.0.tar.gz
+    md5: 06258afcfcdd2f829167450fd5fed604
+
+build:
+    number: 0
```

Comment:
add 
   skip: True  # [win]
here as in:
https://github.com/ioos/conda-recipes/blob/master/nco/meta.yaml#L9-L11